python/mxnet/rnn/rnn_cell.py

```diff
@@ -404,7 +409,7 @@ def __call__(self, inputs, states):
                                           name="%sslice"%name)
         in_gate = symbol.Activation(slice_gates[0], act_type="sigmoid",
                                     name='%si'%name)
-        forget_gate = symbol.Activation(slice_gates[1], act_type="sigmoid",
+        forget_gate = symbol.Activation(slice_gates[1] + self._forget_bias, act_type="sigmoid",
```

Comment:
Thanks for the suggestion, I'll send a CR with the initializer instead and I'll try to add to FusedRNNCell (its done in TF).
In the long run, do we plan to have identical features of LSTMCell in FusedRNNCell? We may be limited by the API of the Cudnn symbol.RNN when including other LSTM functionalities, maybe we could add another LSTM with richer set of functionalities that is not compatible with FusedRNNCell?